web/satellite/src/components/header/projectSelection/ProjectDropdown.vue

```diff
@@ -100,6 +105,13 @@ export default class ProjectDropdown extends Vue {
         return this.$store.getters.selectedProject;
     }
 
+        /**
+		 * Opens new project creation popup.
+		 */
+        public toggleSelection(): void {
+                this.$store.dispatch(APP_STATE_ACTIONS.TOGGLE_NEW_PROJ);
```

Comment:
I've removed new project popup entirely here https://review.dev.storj.io/c/storj/storj/+/2548. 
We'll have separate page now. So here will be redirect to a new page instead of action dispatch